app/src/docs/v1.api-spec.yaml

```diff
@@ -169,32 +169,19 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
-  "/keyCloak/clients/{keycloakEnv}/{acronym}":
+  "/keyCloak/clients":
```

Comment:
Let's change this to /serviceClients/clients as we want to 1) only focus on service clients (as per discussion) and 2) we want to try to avoid referring to keycloak as much as possible because keycloak is implementation specific.